api/src/main/java/org/pmiops/workbench/notebooks/LeonardoNotebooksClientImpl.java

```diff
@@ -131,26 +133,34 @@ private LeonardoCreateRuntimeRequest buildCreateRuntimeRequest(
             .welderRegistry(WelderRegistryEnum.DOCKERHUB)
             .customEnvironmentVariables(customEnvironmentVariables);
 
+    request.setRuntimeConfig(buildRuntimeConfig(clusterOverride, runtime));
+
+    return request;
+  }
+
+  private Object buildRuntimeConfig(@Nullable ClusterConfig clusterOverride, Runtime runtime) {
```

Comment:
opt: generally I think it's more readable to put the primary param first, followed by any optional / nullable / less-important params